### PR TITLE
Department summary - Add Average of Code Coverage and Fix See Repository

### PR DESCRIPTION
**Pull request by lauperalti**
## What does this PR do?
In View by department:

- Add average of code coverage of a department
Resolves [#EM-118](https://rootstrap.atlassian.net/browse/EM-118)

- Fix error in See Repository button
Resolves [#EM-251](https://rootstrap.atlassian.net/browse/EM-251)

Screen Shot 2021-09-10 at 16 17 46

### CODE REVIEW

File: app/services/code_climate/repositories_summary_service.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def call
         invalid_issues_count_average: invalid_issues_count_average.round(2),
         wont_fix_issues_count_average: wont_fix_issues_count_average.round(2),
         open_issues_count_average: open_issues_count_average.round(2),
+        test_coverage_average: test_coverage_average&.round,
```
Comment (santib) [inline app/services/code_climate/repositories_summary_service.rb:18]: why not `round(2)` as the ones above? Just to be consistent I guess..

File: app/views/development_metrics/repositories.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <div class="row">
   <div class="col-md-12 text-center mt-2">
-    <b><%= @repository.product[:name] %></b>
+    <b><%= @repository.product[:name] if @repository.product.present? %></b>
```
Comment (santib) [inline app/views/development_metrics/repositories.erb:5]: how is this change related to code coverage? usually I'd recommend creating separate PRs for unrelated changes, but we can leave it this time

**Review by santib — APPROVED** [review]
Good job

**Review by enzofab91 — APPROVED** [review]

File: app/views/development_metrics/repositories.erb (hunk 2 above)
Comment (lauperalti) [reply app/views/development_metrics/repositories.erb:5]: you are right! 
It is not related, I found this bug when navigating the web department report by repository and fix it, as it was one line I commit it with this issue. My bad.

Comment (lauperalti) [reply app/views/development_metrics/repositories.erb:5]: As I will leave it this time, I fixed the PR description to have quick reference for this change.

File: app/services/code_climate/repositories_summary_service.rb (hunk 1 above)
Comment (santib) [reply app/services/code_climate/repositories_summary_service.rb:18]: ahh that's a good point, I didn't realize that the other where counts and this one a percentage. I'm fine leaving it like this then 👌

**Pull request merged**